tests/sitemaps_tests/test_http.py

```diff
@@ -252,6 +261,20 @@ def test_empty_sitemap(self):
         response = self.client.get('/empty/sitemap.xml')
         self.assertEqual(response.status_code, 200)
 
+    def test_simple_not_i18n_sitemap_index(self):
+        """
+        A simple non-i18n sitemap index can be rendered.
+        """
+        with self.assertNoLogs('django.template', 'DEBUG'):
+            response = self.client.get('/i18n/not-i18n.xml')
+        expected_content = """<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+<url><loc>{0}/en/i18n/testmodel/{1}/</loc><changefreq>never</changefreq><priority>0.5</priority></url>
+</urlset>
+""".format(self.base_url, self.i18n_model.pk)
+        print(response.content.decode())
+        self.assertXMLEqual(response.content.decode(), expected_content)
```

Comment:
Do we need a new test and `SimpleI18nSitemap`? It should be enough to add a `assertNoLogs()` assertion to the `test_simple_i18n_sitemap_index`, e.g.:
```diff --git a/tests/sitemaps_tests/test_http.py b/tests/sitemaps_tests/test_http.py
index b546c87fe6..566252f28b 100644
--- a/tests/sitemaps_tests/test_http.py
+++ b/tests/sitemaps_tests/test_http.py
@@ -257,7 +257,8 @@ class HTTPSitemapTests(SitemapTestsBase):
         """
         A simple i18n sitemap index can be rendered.
         """
-        response = self.client.get('/simple/i18n.xml')
+        with self.assertNoLogs('django.template', 'DEBUG'):
+            response = self.client.get('/simple/i18n.xml')
         expected_content = """<?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
 <url><loc>{0}/en/i18n/testmodel/{1}/</loc><changefreq>never</changefreq><priority>0.5</priority></url><url><loc>{0}/pt/i18n/testmodel/{1}/</loc><changefreq>never</changefreq><priority>0.5</priority></url>

```